pkgs/development/tools/database/webdis/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, hiredis, http-parser, jansson, libevent }:
+
+stdenv.mkDerivation rec {
+  pname = "webdis";
+  version = "0.1.7";
+
+  src = fetchFromGitHub {
+    owner = "nicolasff";
+    repo = pname;
+    rev = version;
+    sha256 = "1hwdmasdr09zlm2pw91wygc87xq2gyg8mi0lb8yqm5nimw5vnzpq";
+  };
+
+  buildInputs = [ hiredis http-parser jansson libevent ];
+
+  makeFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
```

Comment:
Isn't this enabled by default? If not, please explain in a comment what's the reason to adding this.

Reply:
No, it's not the default. This is the output that comes when line 16 is commented out:

```
gcc -o webdis webdis.o cmd.o worker.o slog.o server.o acl.o md5/md5.o sha1/sha1.o http.o client.o websocket.o pool.o conf.o formats/json.o formats/raw.o formats/common.o formats/custom-type.o hiredis/hiredis.o hiredis/sds.o hiredis/net.o hiredis/async.o hiredis/read.o hiredis/dict.o jansson/src/dump.o jansson/src/error.o jansson/src/hashtable.o jansson/src/load.o jansson/src/strbuffer.o jansson/src/utf.o jansson/src/value.o jansson/src/variadic.o http-parser/http_parser.o b64/cencode.o -levent -pthread
installing
install flags: SHELL=/nix/store/1iaxkm0941nj1m4m5g4fxgg4cq5jckf0-bash-4.4-p23/bin/bash install
mkdir -p //usr/local
mkdir: cannot create directory '//usr': Permission denied
make: *** [Makefile:45: //usr/local] Error 1
builder for '/nix/store/d1sqj80cxlb5vcqssda2clkyr3pwycbr-webdis-0.1.7.drv' failed with exit code 2
error: build of '/nix/store/d1sqj80cxlb5vcqssda2clkyr3pwycbr-webdis-0.1.7.drv' failed
````

I'm not sure what said comment should look like?